frontend/src/scenes/session-recordings/player/controller/PlayerControllerTime.tsx

```diff
@@ -44,10 +47,14 @@ export function SeekSkip({ direction }: { direction: 'forward' | 'backward' }):
 
     const altKeyHeld = useKeyHeld('Alt')
     const jumpTimeSeconds = altKeyHeld ? 1 : jumpTimeMs / 1000
-    const altKeyName = navigator.platform.includes('Mac') ? '⌥' : 'Alt'
 
-    const arrowSymbol = direction === 'forward' ? '→' : '←'
-    const arrowName = direction === 'forward' ? 'right' : 'left'
+    const arrowKey: Partial<Record<HotKeyOrModifier, true>> = {}
+    if (direction === 'forward') {
+        arrowKey.arrowright = true
+    }
+    if (direction === 'backward') {
+        arrowKey.arrowleft = true
+    }
```

Comment:
I can see the `react-scan` thingy flashing in horror in front of my eyes, lol. I suggest using `useMemo` :)

Reply:
It'd be ideal if we could simply say something like `arrowRight={direction === 'forward'}` in the `KeyboardShortcut` component, but we'd need to tweak the inner implementation to achieve that